src/attr.rs

```diff
@@ -231,13 +290,19 @@ impl<I: Iterator<Item = char>> Parser<I> {
                         Invalid
                     }
                 }
-                ValueQuoted => {
-                    if c == '"' {
-                        Whitespace
-                    } else {
+                ValueQuoted => match c {
+                    '\\' => {
+                        if let Some(c) = self.chars.next() {
```

Comment:
I think it would be preferable to add a new state here to the FSM instead of adding another place where we consume a char. Easier to keep track of a state machine where each step is a single char.

Reply:
Or actually, it might make things more complicated due to how the next/step functions work right now. Maybe best to keep it like this for now.